.expeditor/automate_build.sh

```diff
@@ -42,6 +42,7 @@ HAB_FEAT_OFFLINE_INSTALL=true HAB_FEAT_IGNORE_LOCAL=false HAB_ORIGIN=chef HAB_CA
 
 git clone https://github.com/chef/automate.git
 cd automate
+git checkout ssudheer/ruby_update_3.1
```

Comment:
Is this a temporary change? Can we make this configurable? Future work ?

Reply:
Made it configurable.